muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/base/material-loader/fields/text-field.tsx

```diff
@@ -198,15 +198,25 @@ export default class TextField extends React.Component<TextFieldProps, TextField
         delete newProps.minWidth;
         delete newProps.className;
         delete newProps.style;
-        return <div className={props.className} style={props.style}>
+        return <span className={props.className} style={props.style}>
           <input {...newProps}/>
-        </div>
+        </span>
       }
     }
 
+    // Lets calculate textfield width based on textfield size option.
+    // If no option size has been set then we use 50 as a default value.
+    // This sets baseline width for normal textfields and autogrow textfields so they appear at same width when they have same size set.
+    const textfieldWidth = this.props.content.columns && parseInt(this.props.content.columns) ? parseInt(this.props.content.columns) * 10 : 50
```

Comment:
The second condition should probably be...
`!isNaN(parseInt(this.props.content.columns))`